pkg/datapath/linux/node.go

```diff
@@ -788,7 +788,7 @@ func (n *linuxNodeHandler) insertNeighbor(ctx context.Context, newNode *nodeType
 			// issued arping after us, as it might have a more recent hwAddr value.
 			return
 		}
-		n.neighLastPingByNextHop[nextHopStr] = time.Now()
+		n.neighLastPingByNextHop[nextHopStr] = now
```

Comment:
If line 771 evaluates to `false`, this will be be an empty time, won't it?

Reply:
Yeah, but then `hwAddr` will be nil too, so this won't be evaluated.

Reply:
Tru dat!